docs/source/blackbox/score_decrease.rst

```diff
@@ -0,0 +1,91 @@
+.. _eli5-decrease-in-score:
+
+Decrease in Score
+=================
+
+eli5 provides a way to compute feature importances for any black-box
+estimator by measuring how score decreases when a feature is not available.
+
+A similar method is described in Breiman, "Random Forests", Machine Learning,
+45(1), 5-32, 2001 (available online at
+https://www.stat.berkeley.edu/%7Ebreiman/randomforest2001.pdf) as
```

Comment:
I didn't read the whole paper, but tried to find something like "mean decrease in accuracy" and failed - could you please tell the page number @kmike ?